CineElorrietaMVC/src/main/java/Modelo/Modelo.java

```diff
@@ -1,5 +1,203 @@
 package Modelo;
 
+import java.awt.Dialog;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Timer;
+
+import javax.swing.JDialog;
+import javax.swing.JList;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+
+import Controlador.Controlador;
+
 public class Modelo {
```

Comment:
Esta clase solo se debería de usar para instanciar el resto de objetos del modelo y si quieres los dos array list. Aparte de eso, una clase con 198 lineas es demasiado larga. Refactorizar el código ubicando estás funcionalidades en otras clases del modelo.

Reply:
Ha vuelto a crecer. 303 lineas. Tenéis que intentar refactorizarlo sin que os de ideas. Crear al menos tres o cuatro clases nuevas de esta clase.